integration-tests/docker/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG JDK_VERSION=8-slim
+ARG JDK_VERSION=8-slim-buster
```

Comment:
So, the docker image seems to work ok for me without any changes, however the k8s integration tests are still failing, so it might be related somehow, I'm not certain if `maven:3-jdk-11-slim` is actually related to the openjdk slim images (the maven dockerfile looks like it downloads openjdk itself)

```
Removing intermediate container 249c6ec65c11
 ---> 4cab348af9bf
Step 8/21 : RUN VERSION=$(mvn -B -q org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate       -Dexpression=project.version -DforceStdout=true     )  && tar -zxf ./distribution/target/apache-druid-${VERSION}-bin.tar.gz -C /opt  && mv /opt/apache-druid-${VERSION} /opt/druid
 ---> Running in ef1f63133a73
ls: cannot access '/usr/bin/mvn': Operation not permitted
Error: Could not find or load main class org.codehaus.plexus.classworlds.launcher.Launcher
Caused by: java.lang.ClassNotFoundException: org.codehaus.plexus.classworlds.launcher.Launcher
The command '/bin/sh -c VERSION=$(mvn -B -q org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate       -Dexpression=project.version -DforceStdout=true     )  && tar -zxf ./distribution/target/apache-druid-${VERSION}-bin.tar.gz -C /opt  && mv /opt/apache-druid-${VERSION} /opt/druid' returned a non-zero code: 1
[ERROR] Command execution failed.
```

I'm also having a bit of trouble running the integration test locally to troubleshoot the issue, the scripts seem to expect Linux.

Reply:
Oops, i was looking at the wrong dockerfile, it is using the openjdk base image https://github.com/carlossg/docker-maven/blob/0a7b1ef030687a5d95ce3e7567a4c279ab87dc81/openjdk-11-slim/Dockerfile#L1

That said, there doesn't seem to be any buster version of this image, related https://github.com/carlossg/docker-maven/issues/221 I think.

At some point we need to like work out whatever the problem is so we can use the newer image.

Reply:
I saw a comment on the linked issue that you can pin to `openjdk:11-jdk-slim-buster` to solve this issue